src/lib.rs

```diff
@@ -1,3 +1,109 @@
+//! # Floem
+//! Floem is cross-platform GUI framework for Rust 🦀. It aims to be extremely performant while providing world-class developer ergonomics.
+//!
+//! ## Views
+//! Floem models the UI using a tree of [Views](view::View)s that are built once. `Views` react to state changes and events.
+//! Views are type-erased, self-contained components that can be composed together to create complex UIs.
```

Comment:
I'm pretty sure that that views are not type erased. Only when using box_container are views cast to dyn view. All the other times they are strongly typed

Reply:
thank you. meant to say they're not generic over their children or state like some other libs. But think we can leave that out.